Correct the Scaling strategies in ScalingJob docs. (#465)
kedacore/keda-docs: content/docs/2.4/concepts/scaling-jobs.md

Select a Scaling Strategy. Possible values are `default`, `custom`, or `accurate`. The default value is `default`.

> 💡 **NOTE:**
>
>`maxScale` is not the running Job count. It is measured through:
 >```go
 >maxValue = min(scaledJob.MaxReplicaCount(), devideWithCeil(queueLength, targetAverageValue))
 >```

>  >that means, pure the value queueLength devided by targetAvarageValue unless it is exceed the MaxReplicaCount.

Reviewer: Did you mean this?
```suggestion
 >That means it will use the value of `queueLength` divided by `targetAvarageValue` unless it is exceeding the `MaxReplicaCount`.
```

>

> >`RunningJobCount` means the number of jobs that are in the running state or not finished. It is measured through:

Reviewer: What do you think of this?
```suggestion
>`RunningJobCount` represents the number of jobs that are currently running or have not finished yet. It is measured as follows:
```

>```go
>if !e.isJobFinished(&job) {
>		runningJobs++
>}
>```

> >`PendingJobCount` means the no. of jobs that are in pending state i.e job which is not

Reviewer: ```suggestion
>`PendingJobCount` provides an indication of the amount of jobs that are in pending state, for example a that has not 
```

> finished **AND** pod with respect to job is either not running or not completed. it is measured through:

Reviewer: ```suggestion
finished yet **and** the underlying pod is either not running or has not completed yet.

It is measured as follows:
```
